### PR TITLE
Traffic Ops - Constraint violation(s) with Traffic Ops postinstall script

### LINKED ISSUE #2718: Traffic Ops - Post Install script fails to add admin user
Post install script fails to add the admin user to the tm_user table (new install).   

```  
ERROR:  null value in column "tenant_id" violates not-null constraint
DETAIL:  Failing row contains (5, admin, null, 1, null, null, SCRYPT:16384:8:1:m+XXXXXXXXX+qAf3SXylI8m2q44NBCMs8aHNaWm3S1oDxhc..., SCRYPT:16384:8:1:m+XXXXXXXXXXXXf3SXylI8m2q44NBCMs8aHNaWm3S1oDxhc..., 2018-08-20 15:35:35.817679-06, null, null, nu    ll, f, null, null, null, null, null, null, null, null, null, null).
SQL state: 23502
```

### PR DESCRIPTION
**Pull request by JBevillC**
Constraint violation(s) with Traffic Ops postinstall script

* admin user doesn't get added to tm_user table due to missing tenant_id
* global profile doesn't get added to profile table due to missing cdn.id 

Fixes #2718 

Original SQL statement ERROR details:

```
ERROR:  null value in column "tenant_id" violates not-null constraint
DETAIL:  Failing row contains (5, admin, null, 1, null, null, SCRYPT:16384:8:1:m+XXXXXXXXX+qAf3SXylI8m2q44NBCMs8aHNaWm3S1oDxhc..., SCRYPT:16384:8:1:m+XXXXXXXXXXXXf3SXylI8m2q44NBCMs8aHNaWm3S1oDxhc..., 2018-08-20 15:35:35.817679-06, null, null, nu    ll, f, null, null, null, null, null, null, null, null, null, null).
SQL state: 23502
```

```
ERROR:  null value in column "cdn" violates not-null constraint
DETAIL:  Failing row contains (15, GLOBAL, Global Traffic Ops profile, DO NOT DELETE, 2018-08-20 16:41:51.641532-06, UNK_PROFILE, null, f).
SQL state: 23502
```

#### Which TC components are affected by this PR?

- [ ] Documentation
- [ ] Grove
- [ ] Traffic Analytics
- [ ] Traffic Monitor
- [X] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] Other _________

#### What is the best way to verify this PR?

1. Run the postinstall script on newly created (empty) postgres DB instance.
2. Verify that the admin user has been successfully added to the tm_user table.
3. Verify that the global profile is in the profile table.

#### Check all that apply

- [ ] This PR includes tests
- [ ] This PR includes documentation updates
- [ ] This PR includes an update to CHANGELOG.md
- [ ] This PR includes all required license headers
- [X] This PR does *NOT* fix a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)

### CODE REVIEW

File: traffic_ops/install/bin/_postinstall [hunk 1]
```diff
@@ -617,8 +617,9 @@ QUERY
     # Skip the insert if the admin 'username' is already there.
     my $hashed_passwd = hash_pass( $adminconf->{"password"} );
     my $insert_admin = <<"ADMIN";
-    insert into tm_user (username, role, local_passwd, confirm_local_passwd)
+    insert into tm_user (username, tenant_id, role, local_passwd, confirm_local_passwd)
                 values  ('$adminconf->{"username"}',
+                        (select id from tenant where name = 'unassigned'),
```
Comment (dangogh) [inline traffic_ops/install/bin/_postinstall]: this is the admin -- should be in the root tenant,  not unassigned

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2300/
Test PASSed.

**Comment by JBevillC** [comment]
Branch updated, squashed, and re-pushed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2301/
Test PASSed.

**Pull request merged**